docs: update client docs for v0.9.2 website credentials login (#30)
Notifiarr/mkdocs-wiki: docs/pages/client/afterInstall.md

## Docker Users

When a new docker image is deployed with an empty `/config` folder mounted, the app will:

- Write a brand new config to `/config/notifiarr.conf` file.
- On first run, the WebUI will prompt you for your API key.
- After entering your API key, login with your **Notifiarr.com email address and password**.

> - You can set a local password after logging in via *Config → Password*.

Reviewer: Config -> Password is a menu tray thing. After you logging in, you set the password in Trust Profile.

> - You can set a local password after logging in via *Trust Profile* (click your username in the top right).

Reviewer: ```suggestion
- You can set a local password after logging in *Trust Profile* by clicking your username in the bottom left.
```

- Write a brand new config to `/config/notifiarr.conf` file.
- On first run, the WebUI will prompt you for your API key.
- After entering your API key, login with your **Notifiarr.com email address and password**.

> - You can set a local password after logging in via *Trust Profile* (click your username in the top right).

Reviewer: bottom right

Reviewer: top left

Reviewer: which is it?!
